roxctl/central/login/login.go

```diff
@@ -262,3 +276,29 @@ You can now use the retrieved access token for all other roxctl commands!
 In case the access token is expired and cannot be refreshed, you have to run "roxctl central login" again.`, centralURL)
 	return nil
 }
+
+func (l *loginCommand) verifyLoginAuthProviders() error {
+	httpClient, err := l.env.HTTPClient(30*time.Second, auth.Anonymous())
+	if err != nil {
+		return errors.Wrap(err, "creating HTTP client")
+	}
+
+	resp, err := httpClient.DoReqAndVerifyStatusCode("/v1/login/authproviders", http.MethodGet,
```

Comment:
maybe add a comment why http and not grpc – AFAIK we do not have anonymous in grpc client